core/tauri-utils/src/config.rs

```diff
@@ -459,6 +459,13 @@ pub struct NsisConfig {
   ///
   /// See <https://github.com/kichik/nsis/tree/9465c08046f00ccb6eda985abbdbf52c275c6c4d/Contrib/Language%20files> for the complete list of languages.
   pub languages: Option<Vec<String>>,
+  /// An key-value pair where the key is the language and the
+  /// value is the path to a custom `.nsi` file that holds the translated text for tauri's custom messages.
+  ///
+  /// See <https://github.com/tauri-apps/tauri/blob/dev/tooling/bundler/src/bundle/windows/templates/nsis-languages/English.nsh> for an example `.nsi` file.
+  ///
+  /// **Note**: the key must be a valid NSIS language and it must be added to [`NsisConfig`]languages array,
+  pub custom_language_files: Option<HashMap<String, PathBuf>>,
```

Comment:
```suggestion
  /// A key-value pair where the key is the language and the
  /// value is the path to a custom `.nsh` file that holds the translated text for tauri's custom messages.
  ///
  /// See <https://github.com/tauri-apps/tauri/blob/dev/tooling/bundler/src/bundle/windows/templates/nsis-languages/English.nsh> for an example `.nsh` file.
  ///
  /// **Note**: the key must be a valid NSIS language and it must be added to [`NsisConfig::languages`] array,
  pub custom_language_files: Option<HashMap<String, PathBuf>>,
```
Not 100% sure about the `::` rustdoc syntax thing.

Reply:
yeah it doesn't work